packages/lwc-language-server/src/lwc-server.ts

```diff
@@ -175,7 +175,18 @@ export default class Server {
     }
 
     private shouldProvideBindingsInHTML(params: CompletionParams): boolean {
-        return params.context?.triggerCharacter === '{';
+        const position = params.position;
```

Comment:
@floralan can you move this logic into another function? I'd like for this to be a little easier to skim and understand. I'm thinking lines 178-188 could all be something different, since you're checking for if char ==='{'. Specifically it would be better to call out, what is char here, and we do we care about it?

Reply:
Yes, this makes sense to me. Just created a new function to it.